src/helpers/token.js

```diff
@@ -1,8 +1,19 @@
 import jwt from 'jsonwebtoken';
-import { config } from 'dotenv';
+import dotenv from 'dotenv';
 import redis from '../database/redis';
 
-config();
+dotenv.config();
+
+const ACCESS_TOKEN_KEY = process.env.TOKEN_SECRET;
+
+export const decodeAcessToken = async (token) => {
+  try {
+    const decoded = await jwt.verify(token, ACCESS_TOKEN_KEY);
+    return decoded;
+  } catch (err) {
+    console.log(err);
+  }
+};
```

Comment:
This function you created is basically repeating `verifyToken` below so it's not needed (you also needed to throw the error caught). Also, you can actually remove the dotenv/config import altogether since it's entry point is in the connection to the database at the start of the application (in _src/database/config/database.js_)